esm/encode.js

```diff
@@ -1,3 +1,5 @@
+const VIEW_CHUNK_SIZE = 2000;
```

Comment:
Please remove this … I need the callback to be embeddable (as in: a toString() should make it portable). The 2000 is an old threshold that ain’t going to change so it can stay where it was.